sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/metadata.yaml

```diff
@@ -1,7 +1,10 @@
 ---
 friendly_name: Clients Daily
-description: A daily aggregate of main pings from each Firefox
-  desktop client, partitioned by day
+description: >-
+  A daily aggregation over `main` pings from each Firefox
+  desktop client, partitioned by day.
+
+  See view: `telemetry.clients_daily`
```

Comment:
Maybe we should be even more explicit about this, something like "This is the backing implementation of `clients_daily`. Normally you would access this via the view `telemetry.clients_daily`" ?

Reply:
Updated to address the spirit of your language here. I also added a note and link about upcoming change to bring in event ping data.